code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -575,3 +589,224 @@
+		qdel(src)
+
+/obj/item/weapon/spellbook/oneuse/teleport
+	spell = /spell/area_teleport
+	spellname = "teleportation"
+	icon_state = "booktele"
+	desc = "This book will really take you places."
+
+/obj/item/weapon/spellbook/oneuse/teleport/recoil(mob/living/carbon/user as mob)
+	if(istype(user, /mob/living/carbon/human))
+		var/mob/living/carbon/human/h = user
+		user.flash_eyes(visual = 1)
+		for(var/datum/organ/external/l_leg/E in h.organs)
+			E.droplimb(1)
+		for(var/datum/organ/external/r_leg/E in h.organs)
+			E.droplimb(1)
+		to_chat(user, "<span class = 'warning'>Your legs fall off!</span>")
+		qdel(src)
+
+/obj/item/weapon/spellbook/oneuse/teleport/blink //sod coding different effects for each teleport spell
+	spell = /spell/aoe_turf/blink
+	spellname = "blinking"
+
+/obj/item/weapon/spellbook/oneuse/teleport/jaunt
+	spell = /spell/targeted/ethereal_jaunt
+	spellname = "jaunting"
+
+/obj/item/weapon/spellbook/oneuse/buttbot
+	spell = /spell/targeted/buttbots_revenge
+	spellname = "ass magic"
+	icon_state = "bookbutt"
+
+/obj/item/weapon/spellbook/oneuse/buttbot/recoil(mob/living/carbon/user as mob)
+	if(istype(user, /mob/living/carbon/human))
+		var/mob/living/carbon/C = user
+		if(C.op_stage.butt != 4)
+			var/obj/item/clothing/head/butt/B = new(C.loc)
+			B.transfer_buttdentity(C)
+			C.op_stage.butt = 4
+			to_chat(user, "<span class='warning'>Your ass just blew up!</span>")
+		playsound(get_turf(src), 'sound/effects/superfart.ogg', 50, 1)
+		C.apply_damage(40, BRUTE, "groin")
+		C.apply_damage(10, BURN, "groin")
+		qdel(src)
+
+/obj/item/weapon/spellbook/oneuse/lightning
+	spell = /spell/lightning
+	spellname = "lightning"
+	icon_state = "booklightning"
+
+/obj/item/weapon/spellbook/oneuse/lightning/recoil(mob/living/carbon/user as mob)
+	if(istype(user, /mob/living/carbon/human))
+		user.apply_damage(25, BURN, "l_hand")
+		user.apply_damage(25, BURN, "r_hand")
+		to_chat(user, "<span class = 'warning'>The book heats up and burns your hands!</span>")
+		qdel(src)
+
+/obj/item/weapon/spellbook/oneuse/timestop
+	spell = /spell/aoe_turf/fall
+	spellname = "time stopping"
+	icon_state = "booktimestop"
+	desc = "A rare, vintage copy of 'WizzWizz's Magical Adventures."
+
+/obj/item/weapon/spellbook/oneuse/timestop/recoil(mob/living/carbon/user as mob)
+	if(istype(user, /mob/living/carbon/human))
+		user.stunned = 5
+		user.flash_eyes(visual = 1)
+		to_chat(user, "<span class = 'warning'>You have been turned into a statue!</span>")
+		new /obj/structure/closet/statue(user.loc, user) //makes the statue
+		qdel(src)
+	return
+
+
+/obj/item/weapon/spellbook/oneuse/timestop/statute //recoil effect is same as timestop effect so this is a child
+	spell = /spell/targeted/flesh_to_stone
+	spellname = "sculpting"
+	icon_state = "bookstatue"
+	desc = "This book is as dense as a rock."
+
+// Spell Book Bundles//
+
+/obj/item/weapon/storage/box/spellbook
+	name = "Spellbook Bundle"
+	desc = "I wonder what magicial items are contained inside?"
```

Comment:
When I look at this line I can't help but read "I wonder what magical realm is contained inside?"